### DISCUSSION TITLE
increase interval/GOP

### DISCUSSION THREAD

**Discussion by ways88** [pedroSG94/RootEncoder#1230] [Q&A] [answered]
Hi Pedro,

i'm using rotation feature in kotlin, inside the ServiceStreaming class in the prepare() method the prepareVideo method is called where it is possible to set the iFrameInterval, this parameter up to how much can be brought, do you have an example?

Thank you.

**Comment by pedroSG94** [comment, marked as answer]
First of all, that value means that you will send a keyframe each that seconds.
The parameter range is from 0-N (where N is any number but more than 20 in my experience is not recommended and could be ignored using a lower value and 0 is all frames are keyframes)

I always recommend a value between 1-5. More than 5 could produce pixelation problems when you are moving the content continually. Lower value is always better but also increase bandwidth usage. 
I think the ideal is use a value of 2.
